docs/ca/dev/preparation-for-going-live.md

```diff
@@ -23,191 +23,108 @@ We've divided the preparation into approximate timeframes, and you can adjust th
+| CLO-12 | Cloud       | Prepare a DNS strategy                                   | Decide how you want customers to access your shop and verify that you control access and can manage the DNS. For example, the domain of your shop is `spryker.com`, but you want customers to access the Storefront at `www.spryker.com`. For details on how to set up DNS, see [Set up DNS](/docs/ca/dev/set-up-dns.html).<br>- Optional: Delegate DNS. To find out how to delegate a DNS name, see [Setting up a custom SSL certificate](/docs/ca/dev/setting-up-a-custom-ssl-certificate.html).<br>- Make sure Transport Layer security (TLS) certificates are provisioned. If you delegate DNS to Spryker, TLS certificates for your endpoints are created automatically. If you want us to create TLS certificates for your endpoints but don't want to delegate your DNS, request the verification of DNS records through the [Support Portal](https://support.spryker.com). If you don't delegate your DNS and want to use your own certificates, provide them to us as described in [Setting up a custom SSL certificate](/docs/ca/dev/setting-up-a-custom-ssl-certificate.html). |                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| CLO-13 | Cloud       | Decide Email sending handling                          | Decide how email sending should be handled. If you want to send emails using Spryker, decide whether you want to use the native mail service shipped with Spryker Cloud Commerce OS or integrate a third-party one. If you want to use the native one, let us know the email address that you want to send emails from. We will lift sending restrictions and help you validate the needed DNS name. For more information about the default email service and its restrictions, see [Email service](/docs/ca/dev/email-service/email-service.html).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
How can I send emails using Spryker by integrating a third-party mail service?

Reply:
By integration a 3rd party service :) 

not sure I understand the question

Reply:
What are you referring to as "sending emails using Spryker"? What's the difference between ""sending emails using Spryker" and "native mail service shipped with Spryker"?

Reply:
Spryker system can trigger 3rd party service or use a native AWS SES mail service